stack.yaml

```diff
@@ -101,12 +104,12 @@ packages:
 # Version of "dns" with proper Windows & Unix multiple-nameservers support
 - location:
     git: https://github.com/kazu-yamamoto/dns.git
-    commit: b106470f0a93672af22cbc7ed6564b53c0f249ed
+    commit: 18ebc30692c71f27746a9faf06c6fc78e19bbe7b
   extra-dep: true
 
 - location:
-    git: https://github.com/well-typed/canonical-json.git
-    commit: 2d261bb971bada1893753b503452d9e6e217bc4a
+    git: https://github.com/angerman/canonical-json.git
+    commit: 37f8140af7b72c794c6293193433fd6dfab7b023
```

Comment:
@dcoutts FYI

Reply:
fwiw, I wonder if `hackage-security` should depend on `canonical-json` just so that `canonical-json` doesn't end up bitrotting?